sdk/open_api/configuration.py

```
@@ -496,7 +496,7 @@ def to_debug_report(self) -> str:
         return "Python SDK Debug Report:\n"\
                "OS: {env}\n"\
```

Ruff (0.15.5)

[warning] Implicitly concatenated string literals over multiple lines

(ISC002)